x-pack/legacy/plugins/infra/public/containers/logs/log_summary/log_summary.test.tsx

```diff
@@ -34,50 +37,43 @@ describe('useLogSummary hook', () => {
   it.skip('queries for new summary buckets when the source id changes', async () => {
     const firstMockResponse = createMockResponse([{ start: 99000, end: 101000, entriesCount: 1 }]);
     const secondMockResponse = createMockResponse([{ start: 99000, end: 101000, entriesCount: 2 }]);
-    const mockApolloClient = {
-      query: jest
-        .fn()
-        .mockResolvedValueOnce(firstMockResponse)
-        .mockResolvedValueOnce(secondMockResponse),
-    };
+
+    fetchLogSummaryMock
+      .mockResolvedValueOnce(firstMockResponse)
+      .mockResolvedValueOnce(secondMockResponse);
 
     const { act, getLastHookValue } = mountHook(
       ({ sourceId }) => useLogSummary(sourceId, 100000, 1000, null),
-      createMockApolloProvider(mockApolloClient),
+      Identity,
       { sourceId: 'INITIAL_SOURCE_ID' }
     );
-
-    expect(mockApolloClient.query).toHaveBeenCalledTimes(1);
-    expect(mockApolloClient.query).toHaveBeenLastCalledWith(
-      expect.objectContaining({
-        variables: expect.objectContaining({
-          sourceId: 'INITIAL_SOURCE_ID',
-        }),
-      })
-    );
-    expect(getLastHookValue().buckets).toEqual(
-      firstMockResponse.data.source.logSummaryBetween.buckets
-    );
+    expect(fetchLogSummaryMock).toHaveBeenCalledTimes(1);
+    // expect(mockApolloClient.query).toHaveBeenLastCalledWith(
+    //   expect.objectContaining({
+    //     variables: expect.objectContaining({
+    //       sourceId: 'INITIAL_SOURCE_ID',
+    //     }),
+    //   })
+    // );
```

Comment:
This is probably a left-over?

Reply:
No. I was waiting to know what I was supposed to do with the source. Now I know and I can test it properly